### PR TITLE
fix E2XX and E3XX flake8 rules and ignore W503

### PR DESCRIPTION
**Pull request by noviluni**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 # directory, add these directories to sys.path here. If the directory is
```
Comment (Gallaecio) [inline docs/conf.py:20]: In this file, I would instead all the sections with commented-out code.

They exist to make the initial configuration file creation easy, but they are not needed after that, and they can even be misleading (as you could think these are all the settings there are, but there may be upstream changes that add, remove or change the default values of some of these settings).

File: pytest.ini [hunk 2]
```diff
@@ -7,6 +7,8 @@ addopts =
     --ignore=dateparser/calendars/hijri_parser.py
     --ignore=dateparser/calendars/hijri.py
 flake8-ignore =
+    W503
```
Comment (Gallaecio) [inline pytest.ini:10]: Maybe include a comment to indicate why it is ignored?

File: tests/test_freshness_date_parser.py [hunk 3]
```diff
@@ -252,7 +252,7 @@ def now_with_timezone(self, tzinfo):
         param('2 tuần 3 ngày', ago={'weeks': 2, 'days': 3}, period='day'),
         # following test unsupported, refer to discussion at:
         # http://github.com/scrapinghub/dateparser/issues/33
-        #param('1 năm 1 tháng 1 tuần 1 ngày 1 giờ 1 chút',
+        # param('1 năm 1 tháng 1 tuần 1 ngày 1 giờ 1 chút',
```
Comment (Gallaecio) [inline tests/test_freshness_date_parser.py:255]: Maybe we should extract this into an `xfail`-marked test method.

File: docs/conf.py (hunk 1 above)
Comment (noviluni) [reply docs/conf.py]: agree!

File: pytest.ini (hunk 2 above)
Comment (noviluni) [reply pytest.ini:12]: Done!

File: tests/test_freshness_date_parser.py (hunk 3 above)
Comment (noviluni) [reply tests/test_freshness_date_parser.py:255]: There are other similar cases (commented tests) and I prefer not to fix all of them in this PR. If you want we can open a new issue to handle this.

**Pull request merged**